assets/multiqc_config.yml

```diff
@@ -0,0 +1,38 @@
+report_comment: >
+  This report has been generated by the <a href="https://github.com/nf-core/nascent" target="_blank">nf-core/nascent</a>
+  analysis pipeline. For information about how to interpret these results, please see the
+  <a href="https://nf-co.re/nascent" target="_blank">documentation</a>.
+report_section_order:
+  "nf-core-nascent-methods-description":
+    order: -1000
+  software_versions:
+    order: -1001
+  "nf-core-nascent-summary":
+    order: -1002
+
+run_modules:
+  - custom_content
+  - fastqc
+  - fastp
+  - samtools
+  - preseq
+  - rseqc
+  - featureCounts
+  - homer
```

Comment:
If these tools run, should also maybe have them listed in the docs - which are currently a bit sparse :-)

Reply:
They're all in output, I missed updating the Pipeline summary, but that should be up-to-date now!